monai/bundle/scripts.py

```diff
@@ -395,8 +397,10 @@ def load(
     workflow_name: str | BundleWorkflow | None = None,
     args_file: str | None = None,
     copy_model_args: dict | None = None,
+    return_state_dict: bool = True,
+    net_override: dict = {},
```

Comment:
there's an error issue with this mutable default, I think it could default to `None` and in the function add something like `if net_override is None: net_override = {}`

Reply:
Thanks, have updated it to `None`.